content/en/docs/languages/php/context.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Context
+weight: 55
+cSpell:ignore: Swoole
+description: Learn how context works in instrumented applications.
+---
+
+OpenTelemetry works by storing and propagating telemetry data. For example, when
+an instrumented application receives a request and a span starts, the span must
+be available to a component which creates child spans. To address this need,
+OpenTelemetry stores the span in the context.
+
+See the [Context specification](/docs/specs/otel/context/) for more information
+on what the context is and does.
```

Comment:
Hmm, maybe it would actually make more sense to link to either of these targets in the **Concepts** section?

- https://deploy-preview-3857--opentelemetry.netlify.app/docs/concepts/context-propagation/
- https://deploy-preview-3857--opentelemetry.netlify.app/docs/concepts/context-propagation/#context

Reply:
Possibly both. The spec page I've linked to defines what the Context API should do (and this page is really talking about our implementation of that API). Context propagation has its own page, https://deploy-preview-3857--opentelemetry.netlify.app/docs/languages/php/propagation/
Perhaps this page really needs to clarify that it's talking about the Context API...

Reply:
I decided to just remove the sentence. It's not really related to context propagation, but nor does the average user need to read the spec.